app/src/main/kotlin/com/wire/android/datastore/GlobalDataStore.kt

```diff
@@ -89,6 +89,6 @@ class GlobalDataStore @Inject constructor(@ApplicationContext private val contex
      * If there is no status stored, the status will be [UserMigrationStatus.NoNeed]
      * meaning that the user does not need to be migrated.
      */
-    fun getUserMigrationStatus(userId: String): Flow<UserMigrationStatus> =
-        context.dataStore.data.map { UserMigrationStatus.fromInt(it[userMigrationStatusKey(userId)] ?: UserMigrationStatus.NoNeed.value) }
+    fun getUserMigrationStatus(userId: String): Flow<UserMigrationStatus?> =
```

Comment:
this change will make the app return null on `ShouldTriggerMigrationForUserUserCase` which will trigger migration again